src/js/visual/configurator.js

```diff
@@ -46,9 +46,13 @@ ripe.Configurator.prototype.init = function() {
     // highlight operation (considers only the default ones)
     this.partsList = [];
     this.owner.getConfig(function(config) {
-        var defaults = config.defaults;
         this.hiddenParts = config.hidden;
-        this.partsList = Object.keys(defaults);

```

Comment:
this was the issue as the order was wrong right ?

Reply:
No, the parts list is in alphabetic order. The problem was that the defaults contains hidden parts like the patch. And on Regent's case it as ```metal_caps```which no longer exists.

Reply:
this should definitly not pass the ripe building process, I need to create a verification for this ...